bin/TombRaidervr_actions.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
Not that I know of. GetControllerState used to work, but it seems to have been deprecated in favor of external json files. If we want to use the legacy control system, we will need to use an older version of OpenVR.DLL and OpenVR.h. I'm in favor of the new system since it should theoretically work for all current controllers and all future controllers that are supported by OpenVR. Also it gives players the option to bind the controls any way they want.